src/validator/validator.ts

```diff
@@ -1,26 +1,31 @@
+import * as airnodeValidator from '@api3/airnode-validator';
+import * as airnodeOis from '@api3/ois';
 import { z, ZodFirstPartySchemaTypes } from 'zod';
 
 export const templateParametersSchema = z.object({ type: z.string(), name: z.string(), value: z.string() });
 
-export const triggerSchema = z.object({
+export const rrpBeaconServerKeeperJobsTriggerSchema = z.object({
   chainIds: z.array(z.string()),
-  templateId: z.string(),
+  templateId: airnodeValidator.config.evmIdSchema,
   templateParameters: z.array(templateParametersSchema),
-  endpointId: z.string(),
+  endpointId: airnodeValidator.config.evmIdSchema,
   deviationPercentage: z.string(),
   keeperSponsor: z.string(),
   requestSponsor: z.string(),
 });
 
+// TODO: XOR?
```

Comment:
XOR sounds like the thing we want, right?

Reply:
I never got an "official" answer. I just put that comment in there because I saw Airkeeper being used with only one kind of triggers at a time but I'm not 100% there isn't a case where we might need Airkeeper to update beacons using both trigger types. Or they might just need to be optional 🤷🏻 

@aquarat any thoughts?

Reply:
I think the use of RRP for beacons is very unlikely in practice and they wouldn't be used by API3 in production. AFAIK they were a demonstration of the power of RRP as a tech stack component.

The only scenario I can think of where RRP beacons would be useful is where you have an API provider with an existing Airnode who is unlikely or unwilling to support PSP directly. Why would they not want to support PSP directly? Maybe they don't want their data _streamed_/made public/resold, in which case supporting it indirectly via RRP would probably displease them eventually. The idea is that Airkeeper and Airnode will eventually become one application, which will make RRP beacons even less likely.

Would there be a negative impact to supporting both protocols concurrently? Is it just a reserved concurrency question? It looks like Airkeeper doesn't care. If the impact is not negative then I'd go with OR (either or both). If there's the potential for complications down the line that could cost more time I'd make it XOR.

I think the main thing is that this is relatively unimportant, so feel free to pick one and go with it. 👍 🤷